modules/lwjgl/core/src/main/java/org/lwjgl/system/APIUtil.java

```diff
@@ -80,10 +80,9 @@ private APIUtil() {
      *
      * @param msg the message to print
      */
-    public static void apiLog(@Nullable CharSequence msg) {
+    public static void apiLog(CharSequence msg) {
```

Comment:
Not sure why I marked this as `@Nullable` in the first place. While the method could technically accept `null`, we probably don't ever want to do that.